Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -1003,4 +1003,4 @@ extension TabTrayController: ClientPickerViewControllerDelegate {
     func clientPickerViewControllerDidCancel(clientPickerViewController: ClientPickerViewController) {
         clientPickerViewController.dismissViewControllerAnimated(true, completion: nil)
     }
-}
\ No newline at end of file
+}
```

Comment:
unrelated change but I was originally in the file and the newline at EOF should be added anyway. let me know if you want me to remove this change though.